packages/block-directory/src/store/test/actions.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * WordPress dependencies
+ */
+import * as BlockFunctions from '@wordpress/blocks';
```

Comment:
Upper case variable names usually imply that it's a class or a React component. I'd suggest `blockFunctions` here.